<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMDB Categorized</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imdbResources.qrc">
    <normaloff>:/images/movie</normaloff>:/images/movie</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMovies">
    <property name="title">
     <string>Movies</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMovies"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/prefs</normaloff>:/images/prefs</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/movie</normaloff>:/images/movie</iconset>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
   <property name="toolTip">
    <string>Scan and Categorize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRefresh_Internal_Movie_List">
   <property name="text">
    <string>Refresh Internal Movie List</string>
   </property>
   <property name="toolTip">
    <string>Refresh Movie List</string>
   </property>
  </action>
  <action name="actionView_Logs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/logs</normaloff>:/images/logs</iconset>
   </property>
   <property name="text">
    <string>View Logs</string>
   </property>
   <property name="toolTip">
    <string>View Logs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionReclaim_Space">
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/recycle</normaloff>:/images/recycle</iconset>
   </property>
   <property name="text">
    <string>Reclaim Space</string>
   </property>
   <property name="toolTip">
    <string>Reclaim Space</string>
   </property>
  </action>
  <action name="actionView_Remaining_Movies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/fix</normaloff>:/images/fix</iconset>
   </property>
   <property name="text">
    <string>View Remaining Movies (Manual Fix)</string>
   </property>
   <property name="toolTip">
    <string>View Remaining Movies</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/Stop</normaloff>:/images/Stop</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Scan</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionView_Status">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/info</normaloff>:/images/info</iconset>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
   <property name="toolTip">
    <string>View Status</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionPush_Que">
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/Push</normaloff>:/images/Push</iconset>
   </property>
   <property name="text">
    <string>Jump Que</string>
   </property>
   <property name="toolTip">
    <string>Jump Que</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionRefresh_Categories">
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/Categorise</normaloff>:/images/Categorise</iconset>
   </property>
   <property name="text">
    <string>Refresh Categories</string>
   </property>
   <property name="toolTip">
    <string>Refresh Categories</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionMissing_Gallery_Pictures">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="imdbResources.qrc">
     <normaloff>:/images/photos</normaloff>:/images/photos</iconset>
   </property>
   <property name="text">
    <string>Missing Gallery Pictures</string>
   </property>
   <property name="toolTip">
    <string>Missing Gallery Pictures</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imdbResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
